MVP/README.md

```diff
@@ -0,0 +1,76 @@
+# Platform-CI MVP
+
+## What's MVP?
+
+Minimal Viable Product (MVP) is a minimized set of Jenkins job definition (and supporting files) on Platform CI:
+
+  - for End-to-End workflow changes (batch update) via **Template / Default YAML**
+  - for create/update/scale tests (by individuals) via **Test / Job YAML**
+
+## How to create a CI job with MVP?
+
+1. ####Get MVP files
+
+  >$ git clone https://github.com/RHQE/platform-ci.git
+
+  >$ cd platform-ci/MVP
+
+1. ####Tweak [sample_job.yaml](/MVP/sample_job.yaml/) with your own test parameters
+
+    *Job YAMLs store parameters which are specific for your testing jobs. It's recommended to maintain them in your team git repo.*
+
```

Comment:
I am not sure how the above plan is expected to scale post MVP. Do you have a plan ?